src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -372,6 +373,29 @@ public void ILLink_verify_analysis_warnings_hello_world_app(string targetFramewo
             Assert.True(!missingWarnings.Any() && !extraWarnings.Any(), errorMessage);
         }
 
+        [Theory]
+        [InlineData("net5.0")]
```

Comment:
Since `net5.0` has already shipped, Should we be considering this a breaking change when users start building their existing `net5.0` apps with the 6.0 SDK?

Reply:
This is great point - we should probably only change the default for `net6.0` and above. Everything else can stay the same, just changing the default value of the feature switch when trimming should only happen for `net6.0` apps and above.